kurtosis-devnet/justfile

```diff
@@ -0,0 +1,29 @@
+set shell := ["/bin/bash", "-c"]
+
+# Internal recipes for kurtosis-devnet
+_contracts-build BUNDLE='contracts-bundle.tar.gz':
```

Comment:
Thoughts on putting this in `contracts-bedrock` rather than a new `kurtosis-devnet` package?